### PR TITLE
Add ceph-csi how-to (Juju)

### PR DESCRIPTION
**Pull request by petrutlucian94**
We're adding a guide that describes the steps to deploy Ceph and the Ceph CSI plugin using Juju.

It mostly follows the steps outlined in the ceph-csi charm readme: https://github.com/charmed-kubernetes/ceph-csi-operator/blob/main/README.md

### CODE REVIEW

File: docs/src/charm/howto/ceph-csi.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+If Juju is configured to use the localhost/LXD cloud, please add
+the following constraint to the osd and k8s units:
+``virt-type=virtual-machine``. The k8s unit should then also be configured to
+use a VM container with ubuntu 22.04 as base:
+
+```
+juju deploy k8s --channel=latest/edge \
+    --base "ubuntu@22.04" \
+    --constraints "cores=2 mem=8G root-disk=16G virt-type=virtual-machine"
```
Comment (nhennigan) [inline docs/src/charm/howto/ceph-csi.md:41]: The prerequisite states you already have a canonical kubernetes cluster running. I would add it as a prerequisite that if running with localhost/LXD  the flag virt-type=virtual-machine be added to the k8s VM container

**Comment by petrutlucian94** [comment]
> In order for this page to be accessible from the navigation, you need to add it to the index page at docs/src/charm/howto/index.md

Whops, forgot to add that. I've updated the PR, thanks!

**Comment by nhennigan** [comment]
I made some changes to your PR. It was mainly styling. Once you have a read through to make sure it still all makes sense go ahead and merge. Great work :smile:

**Review by nhennigan — APPROVED** [review]

**Review by petrutlucian94 — COMMENTED** [review]
@nhennigan thanks for making these changes! I think we should keep the ``juju deploy k8s`` example, maybe even add an explicit note about the vm resources. By default, it will allocate a single vm cpu, which isn't enough and some k8s services and pods will hang.

File: docs/src/charm/howto/ceph-csi.md [hunk 2]
```diff
@@ -1,34 +1,24 @@
...
-This guide assumes that you have an already existing {{product}} cluster.
+This guide assumes that you have an existing {{product}} cluster.
 See the [charm installation] guide for more details.
 
 In case of localhost/LXD Juju clouds, please make sure that the K8s units are
-configured to use VM containers with ubuntu 22.04 as base. For example, use
-the following:
-
-```
-juju deploy k8s --channel=latest/edge \
```
[the remark this answers is not in the document]
Comment (nhennigan) [reply docs/src/charm/howto/ceph-csi.md]: I was unaware of the resource importance with that command. I think your latest commit is a good middle ground

**Comment by nhennigan** [comment]
LGTM. You can go ahead and merge. Thanks for all your help

**Pull request merged**